Jellyfin.ApiClient/ApiClient.cs

```diff
@@ -231,43 +193,41 @@ private async Task ValidateConnectionInternal(ConnectionMode initialConnectionMo
                 networkStatus = NetworkConnection.GetNetworkStatus();
             }
 
-            var urlList = new List<Tuple<string, ConnectionMode>>
+            var urlList = new List<Uri>
 			{
-				new Tuple<string,ConnectionMode>(ServerInfo.LocalAddress, ConnectionMode.Local),
-				new Tuple<string,ConnectionMode>(ServerInfo.RemoteAddress, ConnectionMode.Remote)
+				ServerInfo.Address,
 			};
 
             if (!networkStatus.GetIsAnyLocalNetworkAvailable())
             {
                 urlList.Reverse();
             }
 
-            if (!string.IsNullOrEmpty(ServerInfo.ManualAddress))
+            if (!string.IsNullOrEmpty(ServerInfo.Address.ToString()))
             {
-                if (!string.Equals(ServerInfo.ManualAddress, ServerInfo.LocalAddress, StringComparison.OrdinalIgnoreCase) &&
-                    !string.Equals(ServerInfo.ManualAddress, ServerInfo.RemoteAddress, StringComparison.OrdinalIgnoreCase))
+                if (!ServerInfo.Address.ToString().Equals(ServerInfo.Address.ToString(), StringComparison.OrdinalIgnoreCase) &&
+                    !ServerInfo.Address.ToString().Equals(ServerInfo.Address.ToString(), StringComparison.OrdinalIgnoreCase))
```

Comment:
Pretty sure this isn't intended

Reply:
Yep, saw that and thought it was weird when checking all the `ToString()`s again. Removed the condition altogether in a13ac6a because it was doing nothing. Not sure what the intended behavior there is so will probably touch it again if I use that code while implementing the client.